components/scream/docs/build.md

```diff
@@ -22,7 +22,7 @@ CMake's testing tool.
 ## 1. Start From a Trustworthy Commit
 
 First, make sure you've cloned the [SCREAM repo](https://github.com/E3SM-Project/scream)
-to `SCREAM_SRC_DIR`. If you're running a branch that's not `master`, check out
+to `SCREAM_SRC_DIR` (including all submodules). If you're running a branch that's not `master`, check out
```

Comment:
While you're at it, you might add the how-to for submodules: `git submodule update --init --recursive`.

Reply:
Done as suggested. Thanks!